tests/backup/backup_resiliency_test.go

```diff
@@ -1558,3 +1558,200 @@ var _ = Describe("{ScaleDownPxBackupPodWhileBackupAndRestoreIsInProgress}", func
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+// NodeCountForLicensing applies label portworx.io/nobackup=true on any worker node and verifies that this worker node is not counted for licensing
+var _ = Describe("{NodeCountForLicensing}", func() {
```

Comment:
Can we create a flle for license tests and add this test there?

Reply:
yes yes..sure

Reply:
Created a separate file